.cbmc-batch/jobs/aws_byte_buf_append/cbmc-batch.yaml

```diff
@@ -1,4 +1,4 @@
 jobos: ubuntu16
-cbmcflags: "--bounds-check;--pointer-check;--function;aws_byte_buf_append_harness;--unwinding-assertions;--unwindset;memcpy.0:33,__builtin___memcpy_chk.0:33"
+cbmcflags:        "--bounds-check;--div-by-zero-check;--float-overflow-check;--nan-check;--pointer-check;--pointer-overflow-check;--signed-overflow-check;--undefined-shift-check;--unsigned-overflow-check;--unwind;1;--unwinding-assertions;--unwindset;memcpy_impl.0:33,memcpy.0:33"
```

Comment:
There is an unnecessary space at the beginning here.

Reply:
These are autogenerated by the makefile.  Figuring out how to modify the makefile to avoid the extra space will be a backlog item.